core/com.b2international.snowowl.core.rest/build.properties

```diff
@@ -3,4 +3,5 @@ output.. = target/classes
 bin.includes = META-INF/,\
                .,\
                WEB-INF/,\
-               site/
+               site/,\
+               WEB-INF/lib/commons-fileupload-1.4.jar
```

Comment:
`WEB-INF/` above includes this file, no need to change.

Reply:
Done!